### ISSUE TITLE
We can't pass the parameters

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/scaffold-command#7]
### Issue

When we pass the parameters, then the `iterative` mode is called

```
scaffold -t rest -g me.snowdrop -i fruit-demo -v 1.0.0-SNAPSHOT -s 2.1.2.RELEASE
? Create from template (y/N) 
```

**Comment by metacosm** [comment]
The behavior is similar to `odo`'s. I can implement a mixed-mode as it would be a more user-friendly way to interact with the tool.

**Comment by cmoulliard** [comment]
I did a quick test of the PR and still get questions ...

![Screenshot 2019-04-29 12 11 47](https://user-images.githubusercontent.com/463790/56889875-068c7980-6a78-11e9-99d6-3bf177fd6cbc.png)

**Comment by metacosm** [comment]
@cmoulliard you still get questions for stuff that needs answers :)

**Comment by cmoulliard** [comment]
Version is passed as parameter

```
./scaffold -t rest -g me.snowdrop -i fruit-demo -v 1.0.0-SNAPSHOT -s 2.1.2.RELEASE
Selected Spring Boot: 2.1.2.RELEASE
? Use 2.1.2.RELEASE supported version (Y/n) 
```

**Comment by metacosm** [comment]
@cmoulliard yes, but you haven't read the text… 😄
The question is whether or not you want to use the supported version, not which version

**Comment by cmoulliard** [comment]
Correct. This is what happens when you don't read or read it too quickly. BTW, we should perhaps rephrase the sentence to `Use the supported version for this release` ?
What is the parameter to be used then to say if we would like to use or not the supported version ? @metacosm
